docs/config.md

```diff
@@ -705,6 +705,16 @@ This is analogous to `model_context_window`, but for the maximum number of outpu
 
 Maximum number of bytes to read from an `AGENTS.md` file to include in the instructions sent with the first turn of a session. Defaults to 32 KiB.
 
+## project_doc_fallback_filenames
+
+Ordered list of additional filenames to look for when `AGENTS.md` is missing at a given directory level. The CLI always checks `AGENTS.md` first; the configured fallbacks are tried in the order provided. This lets monorepos that already use alternate instruction files (for example, `CLAUDE.md`) work out of the box while you migrate to `AGENTS.md` over time.
+
+```toml
+project_doc_fallback_filenames = ["CLAUDE.md", ".exampleagentrules.md"]
+```
+
+When a fallback is used, the TUI displays a warning reminding you that `AGENTS.md` remains the preferred location for Codex-specific instructions because fallback files may reduce model performance.
```

Comment:
This doc is out of date. You may want to put the warning you had in the UI here instead